src/main/java/org/folio/circulation/domain/ResultItemContext.java

```diff
@@ -4,12 +4,12 @@
 
 import java.util.List;
 
-public class HoldShelfClearanceContext {
+public class ResultItemContext {
```

Comment:
What is a result item context? This name does not tell the reader much about why it is used.

Is it an item to be used for generating a report?

Reply:
Renamed. This class is used for getting all items